tests/distributions/test_transformed_distribution.py

```diff
@@ -78,15 +76,15 @@ def make_lognormal(kwargs):
     },
     {
         'example': {
-            'mu': [[1.4, 0.4, 0.4], [1.4, 0.4, 0.6]],
-            'sigma': [[2.6, 0.5, 0.5], [2.6, 0.5, 0.5]],
+            'mu': [1.4, 0.4, 0.4],
+            'sigma': [1.2, 0.5, 0.5],
             'test_data': [[5.5, 6.4, 6.4], [0.5, 0.4, 0.4]],
         },
-        'min_samples': 10000,
+        'min_samples': 500000,
     },
     {
-        'example': {'mu': [[1.4], [0.4]], 'sigma': [[2.6], [0.5]], 'test_data': [[5.5], [6.4]]},
-        'min_samples': 10000,
+        'example': {'mu': [1.4], 'sigma': [1.2], 'test_data': [[5.5], [6.4]]},
+        'min_samples': 1000000,
```

Comment:
I think lognormal is very sensitive to `sigma` so just reducing that and increasing the number of samples.